src/window.rs

```diff
@@ -240,7 +240,7 @@ impl Window {
     /// setting the callback.
     pub fn set_resize_callback<F: Fn(u32, u32) + 'static>(&mut self, func: F) {
         unsafe {
-            RESIZE_CALLBACK = Some(Box::new(func));
+            RESIZE_CALLBACK = Some(Box::into_raw(Box::new(func)));
```

Comment:
I guess this previously dropped the old Box when overwriting it, now it instead leaks. Since that can turn into a memory leak of non-constant size in total, it should probably be addressed.